app/experimenter/nimbus-ui/src/gql/types/GetExperiment.ts

```diff
@@ -0,0 +1,54 @@
+/* tslint:disable */
+/* eslint-disable */
+// @generated
+// This file was automatically generated and should not be edited.
+
+import { NimbusExperimentStatus, NimbusExperimentApplication, NimbusExperimentFirefoxMinVersion, NimbusExperimentTargetingConfigSlug } from "./../../../types/globalTypes";
+
+// ====================================================
+// GraphQL query operation: GetExperiment
+// ====================================================
+
+export interface GetExperiment_experimentBySlug_controlBranch {
+  __typename: "NimbusBranchType";
+  name: string;
+  slug: string;
+  description: string;
+  ratio: number;
+  featureValue: string | null;
+  featureEnabled: boolean;
+}
+
+export interface GetExperiment_experimentBySlug_probeSets {
+  __typename: "NimbusProbeSetType";
+  slug: string;
+  name: string;
+}
+
+export interface GetExperiment_experimentBySlug {
+  __typename: "NimbusExperimentType";
+  name: string;
+  slug: string;
+  status: NimbusExperimentStatus;
+  hypothesis: string | null;
+  application: NimbusExperimentApplication | null;
+  publicDescription: string | null;
+  controlBranch: GetExperiment_experimentBySlug_controlBranch | null;
+  probeSets: GetExperiment_experimentBySlug_probeSets[];
+  channels: (string | null)[];
+  firefoxMinVersion: NimbusExperimentFirefoxMinVersion | null;
+  targetingConfigSlug: NimbusExperimentTargetingConfigSlug | null;
+  populationPercent: number | null;
+  proposedEnrollment: number | null;
+}
+
+export interface GetExperiment {
+  /**
+   * Retrieve a Nimbus experiment by its slug.
+   */
+  experimentBySlug: GetExperiment_experimentBySlug | null;
+}
+
+export interface GetExperimentVariables {
+  slug: string;
+}
```

Comment:
@bbangert this is what the resulting Apollo codegen output looks like, in case you were curious 🙂

Reply:
Ah, interesting.